Sources/TensorFlow/Operators/NN.swift

```diff
@@ -820,3 +820,45 @@ func _vjpAvgPool3D<Scalar: TensorFlowFloatingPoint>(
     }
   )
 }
+
+//===------------------------------------------------------------------------------------------===//
+// Rearrange depth/space
+//===------------------------------------------------------------------------------------------===//
+
+/// Rearranges data from depth into blocks of spatial data.
+///
+/// - Precondition: The input must have rank 4.
+/// - Precondition: The size must be greater than 1.
```

Comment:
All other `Precondition` statements in this repository are English phrasing.
I think it's better to follow that custom in this PR.
There should be a issue to define the rule.

Reply:
Hmm.  I appreciate the desire not to set new precedent and create inconsistency, but  we are trying to improve our API practices, and if we don't start doing some things better before, e.g., a whole-codebase revision of precondition documentation can occur, it will be hard to make progress.

@saeta your thoughts on how we should approach this would be appreciated. 
@dan-zheng shall we add this to the document we have in progress?

Reply:
Adopting common documentation practices sounds great to me!

> @dan-zheng shall we add this to the document we have in progress?

Sounds good!